Formula/libmdbx.rb

```diff
@@ -0,0 +1,34 @@
+class Libmdbx < Formula
+  desc "One of the fastest embeddable key-value ACID database without WAL"
+  homepage "https://erthink.github.io/libmdbx/"
+  url "https://github.com/erthink/libmdbx/releases/download/v0.9.2/libmdbx-amalgamated-0.9.2.tar.gz"
+  sha256 "c35cc53d66d74ebfc86e39441ba26276541ac7892bf91dba1e70c83665a02767"
+  license "OLDAP-2.8"
+
+  TOOLS = %w[mdbx_stat mdbx_copy mdbx_dump mdbx_load mdbx_chk].freeze
+  MANPAGES = TOOLS.map { |tool| "man1/#{tool}.1" }.freeze
+
+  def install
+    system "make", "all"
+
+    bin.install(*TOOLS)
+
+    lib.install "libmdbx.dylib"
+    lib.install "libmdbx.a"
+    include.install "mdbx.h", "mdbx.h++"
+
+    man.mkpath
+    man1.install(*MANPAGES)
```

Comment:
Is there no `make install`?

Reply:
There is but it assumes flags for `install` that don't work on MacOS.

Reply:
Could you submit a patch upstream to make it work on macOS? That's much cleaner than us picking our own install.

Reply:
I don't have time to do so, but maybe someone else can pick up this PR if/when that happens. Closing for now.